src/main/resources/mycila/APACHE-2.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (C) ${project.inceptionYear}-${currentYear} ${owner}
+Copyright (C) ${license.git.copyrightYears} ${owner}
```

Comment:
It should work but depending on the project settings this may update all the headers